doc/user/assets/sass/_layout.scss

```diff
@@ -377,7 +377,6 @@
 
 .sidebar a,
 .toc a {
-    word-wrap: break-word;
```

Comment:
I’m not entirely sure about the scope of the fix. It seems like it applies to the entire documentation. If yes, should we be concerned that this will potentially use more space for the table of content for all pages and leave less space for the actual content?

Reply:
@sthm Thanks for the review it only applies to links within the Table of Contents globally on docs. The main content always stays 840px for larger screens when the screen gets smaller it shrinks the main content as needed to avoid the toc links breaking to a new line or overflowing off of the screen. 

We can't set a max-width on the toc but that will cause overflow issues, right now it is set to `fit-content` which takes however much is needed.

Reply:
Sounds good, thanks!